linux_os/guide/system/auditing/policy_rules/audit_rules_for_ospp/rule.yml

```diff
@@ -43,6 +43,7 @@ identifiers:
 references:
     ospp: FAU_GEN.1.1.c
     nist: AU-2(a)
+    srg: SRG-OS-000004-GPOS-00004,SRG-OS-000240-GPOS-00090,SRG-OS-000241-GPOS-00091,SRG-OS-000303-GPOS-00120,SRG-OS-000476-GPOS-00221,SRG-OS-000327-GPOS-00127,SRG-OS-000064-GPOS-00033,SRG-OS-000365-GPOS-00152,SRG-OS-000458-GPOS-00203,SRG-OS-000461-GPOS-00205,SRG-OS-000462-GPOS-00206,SRG-OS-000463-GPOS-00207,SRG-OS-000465-GPOS-00209,SRG-OS-000466-GPOS-00210,SRG-OS-000467-GPOS-00211,SRG-OS-000468-GPOS-00212,SRG-OS-000470-GPOS-00214,SRG-OS-000471-GPOS-00215,SRG-OS-000471-GPOS-00216,SRG-OS-000472-GPOS-00217,SRG-OS-000474-GPOS-00219,SRG-OS-000475-GPOS-00220,SRG-OS-000477-GPOS-00222
```

Comment:
Nack. @dahaic is responsible for breaking all audit rules out into independent checks.

Reply:
Discussed offline.  I'm checking to see if any audit rules are missing from the files in this OSPP rule.

Reply:
Followup via email, for now.

Reply:
Audit rules are already broken into small pieces on the audit side. https://github.com/linux-audit/audit-userspace/tree/master/rules Please proceed with this mapping, and we will split it when splitting the rule. This PR is huge already, don't wait for us. @yuumasato FYI

Reply:
FYI it IS a requirement to break the audit rules out even further than what was submitted upstream, and this is a hard requirement.

Reply:
Audit rules are already broken into small pieces on the audit side. https://github.com/linux-audit/audit-userspace/tree/master/rules Please proceed with this mapping, and we will split it when splitting the rule. This PR is huge already, don't wait for us. @yuumasato FYI

Reply:
Created https://github.com/ComplianceAsCode/content/issues/5012 and assigned to @dahaic